src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InjectDirective.cs

```diff
@@ -16,7 +16,10 @@ public static class InjectDirective
             DirectiveKind.SingleLine,
             builder =>
             {
-                builder.AddTypeToken().AddMemberToken();
+                builder
+                    .AddTypeToken(Resources.InjectDirective_TypeToken_Name, Resources.InjectDirective_TypeToken_Description)
```

Comment:
@NTaylorMullen - should these by Funcs so they can be localized? Do we do something similar already for directives?

Reply:
Not sure I follow. Funcs in what part? Like `Resources.FormatXYZ(...)` or `.AddTypeToken(builder => builder....)`?

Reply:
like
```
AddTypeToken(Func<string> name, ...)
```

I guess that's not super possible is it since we serialize? We do this trick a bunch of places in MVC

Reply:
Ahh, ya we do that in MVC but that's not possible here sadly for exactly your point, serializability.

Reply:
OK just ignore me then.